pkgs/servers/tidb/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "tidb";
+  version = "6.5.0";
+
+  src = fetchFromGitHub {
+    owner = "pingcap";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-U3RGQADiS3mNq+4U+Qn+LMYbX8vxkTmofnRc+yrAcIA=";
+  };
+
+  vendorSha256 = "sha256-ljPFNjnmPaUx1PHtjlJh9ubKBDS3PgvbqTce9pi3GSc=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/pingcap/tidb/dumpling/cli.ReleaseVersion=${version}"
+    "-X github.com/pingcap/tidb/util/versioninfo.TiDBEdition=Community"
+  ];
+
+  subPackages = [ "tidb-server" ];
+
+  meta = with lib; {
+    description = "TiDB is an open-source, cloud-native, distributed, MySQL-Compatible database for elastic scale and real-time analytics";
```

Comment:
It is wonderful when the "Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md)" box is checked, however the PR fails in the clearest directive:

> meta.description should:
> - Not start with the package name.

Reply:
I fixed it, thanks